src/components/ButtonComponent/index.jsx

```diff
@@ -0,0 +1,6 @@
+import React from "react";
+
+export function Button(props) {
```

Comment:
Let's keep the name of the component and its folder the same.
Folder name "Button" would be enough since it's already in the "components" folder and it's the same as the actual component function name.
We also avoid redundancy in naming this way.
Same goes to other components.

Reply:
All relevant files names have been changed to match the correct naming convention.